internal/sddc/sddc_vsan_subresource.go

```diff
@@ -36,6 +37,11 @@ func GetVsanSchema() *schema.Schema {
 					Description: "VSAN feature Deduplication and Compression flag, one flag for both features",
 					Optional:    true,
 				},
+				"esa_enabled": {
+					Type:        schema.TypeBool,
+					Optional:    true,
+					Description: "Enable vSAN ESA",
```

Comment:
IMHO we need to document when this is possible and any conflicts it may have.

Reply:
Will do, I'll just wait for us to sort out the previous thread, I will probably end up just updating the docs for the affected resources

Reply:
There are no conflicting attributes when doing bringup.
I did add a few words in the docs for clusters though.